targets/TARGET_STMBLUE/DTM/BlueNrgHCIDriver.cpp

```diff
@@ -340,14 +332,19 @@ class TransportDriver : public cordio::CordioHCITransportDriver {
     	  buffer_out[0] = 0x04;
     	  buffer_out[1] = 0x0F;
     	  buffer_out[2] = 0x04;
-    	  buffer_out[3] = 0x01;
+    	  buffer_out[3] = 0x01;      ///01 unknown command <-
     	  buffer_out[4] = 0x01;
     	  Osal_MemCpy(&buffer_out[5], &opCode, 2);
     	  return 7;
     }
 
```

Comment:
@pan- I've found a way to periodically call the BTLE_StackTick and hide this mechanism to the application. I mapped the BTLE_StackTick on an empty vendor specific opcode (0xFCFF) in my HCI command table and a ticker triggers its execution by calling `StackTick()` (below) that in turn calls the HCI wrapper containing BTLE_StackTick. The wrapper is `hci_wrap_BTLE_StackTick_process`, in DTM_cmd_db.c.

Please let me know your opinion.

Reply:
That's a clever hack that is good enough until we provide necessary interfaces in `HCIDriver`. 
I'd like to remove it once the API are presents.

Reply:
@pan- I've found a issue in this approach (it occurs rarely but it is annoying), is there any new concerning your proposed signal mechanism? Alternatively, is there any safe way to push BTLE_StackTick as event inside BLE simple event queue?